core/controllers/story_editor_test.py

```diff
@@ -129,6 +129,30 @@ def test_story_publish_and_unpublish(self):
             expected_status_int=401)
 
 
+class ValidateExplorationsHandlerTests(BaseStoryEditorControllerTests):
+
+    def test_validation_issues(self):
+        # Check that admins can publish a story.
+        self.login(self.ADMIN_EMAIL)
+        self.save_new_valid_exploration(
+            '0', self.admin_id, title='Title 1',
+            category='Mathematics', language_code='en')
+        json_response = self.get_json(
+            '%s/%s/15,0' % (
+                feconf.VALIDATE_STORY_EXPLOATIONS_URL_PREFIX, self.story_id))
+
+        issues = json_response['validation_issues']
+        issue_1 = (
+            'Expected story to only reference valid explorations, but found '
+            'an exploration with ID: 15 (was it deleted?)')
```

Comment:
Surely we would know whether it was deleted or not? Better to make the warning more explicit.

Reply:
The ID could have never existed at all, or could have been deleted, hence the message. 
Should I  use the include_deleted flag to fetch deleted explorations as well and compare to check if the exp was deleted?

Reply:
I think that's probably overkill. Let's just say "but found a reference to an invalid exploration ID: ..." (please pluralize as needed).

Reply:
Done